js/packages/components/items/Text.priv.tsx

```diff
@@ -18,6 +18,7 @@ export const TextPriv: React.FC<{}> = props => {
 
 const styles = StyleSheet.create({
 	container: {
-		width: '75%',
+		paddingRight: 7,
+		flexShrink: 1,
```

Comment:
```suggestion
		width: '90%',
		flexShrink: 1,
```

Hi @mohammed-shakir , your changes still breaks on long strings, eg: pt-br:

<img width="219" alt="Screenshot 2023-11-19 at 09 25 16" src="https://github.com/berty/berty/assets/689440/421e497d-eb64-4a5a-b5e3-8ecb0097a389">

It will work fine 
<img width="210" alt="Screenshot 2023-11-19 at 09 26 43" src="https://github.com/berty/berty/assets/689440/a1543fbe-158d-404d-a310-0d6b74fa9a69">
if you use the 90% width: